dbms/src/Operators/UnorderedSourceOp.h

```diff
@@ -63,6 +62,11 @@ class UnorderedSourceOp : public SourceOp
         return "UnorderedSourceOp";
     }
 
+    void operatePrefix() override
+    {
+        addReadTaskPoolToScheduler();
```

Comment:
Since `addReadTaskPoolToScheduler` is not called during constructor anymore,  there is a chance that destructor is called when `operatePrefix` is not called yet, so I think we can't call `task_pool->decreaseUnorderedInputStreamRefCount();` without extra check in the destructor.

Reply:
It seems that even if addReadTaskPoolToScheduler is not called, it does not affect calling `task_pool->decreaseUnorderedInputStreamRefCount();`.
Only `increaseUnorderedInputStreamRefCount` and `decreaseUnorderedInputStreamRefCount` must be called as a pair.